saleor/static/dashboard-next/AppRoot.tsx

```diff
@@ -376,10 +377,9 @@ export const AppRoot = decorate(
                                       {user.email}
                                     </Typography>
                                     <ArrowDropdown
-                                      className={[
-                                        classes.arrow,
-                                        menuOpen ? classes.rotate : undefined
-                                      ].join(" ")}
+                                      className={classNames(classes.arrow, {
```

Comment:
What about the alternative:

```typescript
classNames({
  [classes.arrow]: true,
  [classes.rotate]: menuOpen
})
```

Reply:
Yeah, I think it would improve readability.